products/mssqlserver.md

```diff
@@ -11,135 +11,408 @@ versionCommand: select @@version
+    latestGdrLink: https://support.microsoft.com/help/5029186
+    latest: "13.0.6435.1 GDR"
     latestReleaseDate: 2023-10-10
     link: https://support.microsoft.com/help/5029186
 
--   releaseCycle: "2014"
+-   releaseCycle: "15.0"
+    codename: Seattle
+    releaseLabel: "2019"
+    releaseDate: 2019-11-04
+    support: 2025-02-28
+    eol: 2030-01-08
+    extendedSupport: false
+    latestGdr: "15.0.2104.1"
+    latestGdrLink: https://support.microsoft.com/help/5029377
+    latest: "15.0.4335.1 CU23"
+    latestReleaseDate: 2023-10-12
+    link: https://learn.microsoft.com/troubleshoot/sql/releases/sqlserver-2019/cumulativeupdate23
+
+-   releaseCycle: "12.0-sp3"
     codename: Hekaton
-    releaseLabel: "2014 SP3 CU4"
+    releaseLabel: "2014 SP3"
     releaseDate: 2018-10-30
     support: 2019-07-09
     eol: 2024-07-09
     extendedSupport: false
-    latest: "12.0.6449.1"
+    latestGdr: "12.0.6179.1"
+    latestGdrLink: https://support.microsoft.com/help/5029184
+    latest: "12.0.6449.1 CU4+GDR"
     latestReleaseDate: 2023-10-10
     link: https://support.microsoft.com/help/5029185
 
--   releaseCycle: "2012"
+-   releaseCycle: "13.0-sp2"
+    codename: SQL16
+    releaseLabel: "2016 SP2"
+    releaseDate: 2018-04-24
+    support: 2022-10-11
+    eol: 2022-10-11
+    extendedSupport: false
+    latestGdr: "13.0.5108.50"
+    latestGdrLink: https://support.microsoft.com/help/5014365
+    latest: "13.0.5893.48 CU17+GDR"
+    latestReleaseDate: 2019-07-09
+    link: https://support.microsoft.com/help/5014351
+
+-   releaseCycle: "11.0-sp4"
     codename: Denali
     releaseLabel: "2012 SP4"
     releaseDate: 2017-10-05
-    support: 2017-07-11
+    support: 2022-07-12
     eol: 2022-07-12
     extendedSupport: 2025-07-08
-    latest: "11.0.7512.11"
+    latestGdr: "11.0.7512.11"
+    latestGdrLink: https://support.microsoft.com/help/5021123
+    latest: "11.0.7512.11 GDR"
     latestReleaseDate: 2023-02-14
     link: https://support.microsoft.com/help/5021123
 
--   releaseCycle: "2008 R2"
+-   releaseCycle: "14.0"
+    codename: Helsinki
+    releaseLabel: "2017"
+    releaseDate: 2017-09-29
+    support: 2022-10-11
+    eol: 2027-10-12
+    extendedSupport: false
+    latestGdr: "14.0.2052.1"
+    latestGdrLink: https://support.microsoft.com/help/5029375
+    latest: "14.0.3465.1 CU31+GDR"
+    latestReleaseDate: 2023-10-10
+    link: https://support.microsoft.com/help/5029376
+
+-   releaseCycle: "13.0-sp1"
+    codename: SQL16
+    releaseLabel: "2016 SP1"
+    releaseDate: 2016-11-16
+    support: 2019-07-09
+    eol: 2019-07-09
+    extendedSupport: false
+    latestGdr: "13.0.4259.0"
+    latestGdrLink: https://support.microsoft.com/help/4505219
+    latest: "13.0.4604.0 CU15+GDR"
+    latestReleaseDate: 2019-07-09
+    link: https://support.microsoft.com/help/4505221
+
+-   releaseCycle: "12.0-sp2"
+    codename: Hekaton
+    releaseLabel: "2014 SP2"
+    releaseDate: 2016-07-14
+    support: 2020-01-14
```

Comment:
Should be Jul 9, 2019, as per https://learn.microsoft.com/en-us/lifecycle/products/sql-server-2014 ?

Reply:
Careful, on https://learn.microsoft.com/en-us/lifecycle/products/sql-server-2014 see Section "Releases", Entry "Service Pack 2" -> Column "End Date" -> Jan 14, 2020

Reply:
I thought the support date was using either the Mainstream End Date, or the Release End Date if it was earlier. If I understand correctly this is only true for the latest service pack (a few SP3 are in this case) ?

Reply:
And in this case is it correct to consider that the _Extended Security Updates_ will be applied to the last service pack ?

Reply:
It is the "End Date" for the releaseCycle (with the SP applied).

> And in this case is it correct to consider that the Extended Security Updates will be applied to the last service pack ?

Yes, that is correct.